tests/jballerina-unit-test/src/test/resources/test-src/types/var/var_with_inferred_type_including_error.bal

```diff
@@ -97,4 +97,18 @@ function testQueryExprsWithError() {
     error? z = from var error(m1, c1) in arr1
                 join var error(m2, c2) in arr2 on val equals 0
                 do {};
- }
+}
+
+function testOnFailClauseWithError() {
```

Comment:
Are we calling this function?

Reply:
Since this is a dataflow analysis test, for the positive tests what I'm testing is that there are no compilation errors for the usage in this file https://github.com/ballerina-platform/ballerina-lang/blob/16fbb7fb26b2b8803b647ae3917d11efaec0ffd9/tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/types/var/VariableDeclaredWithWithVarIncludingErrorTest.java#L38.

This should be sufficient, right?